lib/gooddata/helpers/csv_helper.rb

```diff
@@ -5,6 +5,7 @@
 # LICENSE file in the root directory of this source tree.
```

Comment:
None of the changes in `actions/*.rb` are unit tested. Consider improving the coverage.

Reply:
I evaluated the existing unit tests and they often already cover the logic changed.